### PR TITLE
Replace SIG/WG meeting placeholder references with Zoom links-4

### LINKED ISSUE #4588: Replace SIG/WG meeting placeholder references with Zoom links
Some time ago Zoom did not have sufficient moderation capabilities and the Zoom meeting links were removed from [sigs.yaml] and pointed towards this placeholder:
https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit

As a workaround to help mitigate trolls from causing problems in meetings. This has since been resolved and the entries in [sigs.yaml] should be updated with their correct link.

There are 53 references to the above placeholder as of 2020-03-09. Updated zoom links can be taken from either the SIG/WG's agenda or their calendar event from the [community calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles).

/area community-management
/sig contributor-experience
/help
/good-first-issue

[sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml

### PR DESCRIPTION
**Pull request by AGMETEOR**
**Which issue(s) this PR fixes**:
Fixes #4588

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Some notes
---------
* Cluster API Provider Implementers' office hours (US West Coast) - Zoom link  for “12:00pm PST” is not available

* Regular SIG Meeting in China - Zoom link for “8pm Beijing/5am PST” is not available

* kops Office Hours - Zoom link for “9am PST” not available

File: sigs.yaml [hunk 1]
```diff
@@ -855,97 +855,97 @@ sigs:
...
+    url: https://zoom.us/j/424743530
     archive_url: https://docs.google.com/document/d/1IZ2-AZhe4r3CYiJuttyciS7bGZTTx4iMppcA8_Pr3xE/edit
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
   - description: Cluster API Provider Implementers' office hours (US West Coast)
     day: Tuesday
     time: "12:00"
     tz: PT (Pacific Time)
     frequency: weekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/j/140808484
```
From their agenda 👍

File: sigs.yaml [hunk 2]
```diff
@@ -855,97 +855,97 @@ sigs:
...
+    url: https://zoom.us/j/916523531
     archive_url: https://docs.google.com/document/d/1Gmc7LyCIL_148a9Tft7pdhdee0NBHdOfHS1SAF0duI4/edit
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
   - description: Regular SIG Meeting in China
     day: Tuesday
     time: "20:00"
     tz: Beijing
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/j/179916854
```
ref: https://shimo.im/docs/YcoaYlvRmIYcrwnd

File: sigs.yaml [hunk 3]
```diff
@@ -855,97 +855,97 @@ sigs:
...
     frequency: biweekly
-    url: https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit#
+    url: https://zoom.us/j/612375927
   - description: kops Office Hours
     day: Friday
     time: "09:00"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/my/k8ssigaws
```
ref: https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit#

File: sigs.yaml [hunk 4]
```diff
@@ -855,97 +855,97 @@ sigs:
...
+    url: https://zoom.us/j/179916854
     archive_url: https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo/edit
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
   - description: minikube Office Hours
     day: Monday
     time: "11:00"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: https://zoom.us/j/807524571
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/j/905156508
```

Out of curiosity where did you get this one? I pulled the suggestion from their meeting notes: https://docs.google.com/document/d/1jhfmL1gsgN39uCEgz5pW9tnIotFgHhxq2yfMK3KYE4w/edit#

Comment (AGMETEOR) [reply sigs.yaml]: I was getting all links from the calendar

**Comment by mrbobbytables** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**